docs/upgrading_to_project_factory_v14.0.md

```diff
@@ -0,0 +1,11 @@
+# Upgrading to Project Factory v14.0
+
+The v14.0 release of Project Factory is a backwards compatible release.
+
+### New `random_project_id_string_method` in addition to `random_project_id`
```

Comment:
Since this is no longer breaking, let's move this to faq or troubleshooting docs?